sys/linux/futex.txt

```diff
@@ -17,4 +18,13 @@ robust_list {
 	next	ptr[in, robust_list, opt]
 }
 
+futex_wait_block {
```

Comment:
We don't have a `struct futex_wait_block` anymore, it's named as `struct futex_waitv`.

Reply:
Renamed